lib/omnibus/packager.rb

```diff
@@ -67,13 +68,13 @@ def for_current_system
         family = "solaris"
       end
       if klass = PLATFORM_PACKAGER_MAP[family]
-        klass
+        [klass].flatten
       else
         log.warn(log_key) do
           "Could not determine packager for `#{family}', defaulting " \
           "to `makeself'!"
         end
-        Makeself
+        [Makeself].flatten
```

Comment:
this does the same thing as `klass.is_a?(Array) ? klass : [ klass ]`. it forces the value to an array. and in case it just made a "double array" it flattens it back to a single array.